apps/fz_http/lib/fz_http_web/live/device_live/new_form_component.ex

```diff
@@ -12,6 +13,7 @@ defmodule FzHttpWeb.DeviceLive.NewFormComponent do
   def mount(socket) do
     {:ok,
      socket
+     |> assign(allowed_to_configure?())
```

Comment:
Is this not used? I don't see a `@allow_unprivileged_device_configuration` in the templates.

Reply:
PR is updated :-).

Reply:
Umm what is?

Reply:
https://github.com/firezone/firezone/pull/909/files#diff-2551b8124ba4b1f13a9b5b8601ebbb51fa827bc7bccf2125bbcb7a6df96b9733R90

Reply:
Welp, *now* it's updated :-)

Reply:
Remove the `assign`